vignettes/occds.Rmd

```diff
@@ -59,6 +59,8 @@ data("admiral_ae")
 data("admiral_adsl")
 
 ae <- admiral_ae
+ae <- filter(ae, USUBJID %in% c("01-701-1015", "01-701-1023", "01-703-1086", "01-703-1096", "01-707-1037", "01-716-1024"))
+
```

Comment:
Please put the filtering step in a separate invisible code block as in the `bds_tte.Rmd` file.

Reply:
Change made and pushed.